modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -0,0 +1,81 @@
+== Understanding the Self Node Remediation Operator configuration
+
+The Self Node Remediation Operator creates the `SelfNodeRemediationConfig` CR with the name `self-node-remediation-config` in the Self Node Remediation Operator's namespace. You can edit this CR. However, you cannot create a new CR for the Self Node Remediation Operator.
+
+A change in the `SelfNodeRemediationConfig` CR re-creates the Self Node Remediation daemon set.
+
+The `SelfNodeRemediationConfig` CR resembles the following YAML file:
+
+[source,yaml]
+----
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediationConfig
+metadata:
+  name: self-node-remediation-config
+  namespace: openshift-operators
+spec:
+  safeTimeToAssumeNodeRebootedSeconds: 180 <1>
+  watchdogFilePath: /test/watchdog1 <2>
+  isSoftwareRebootEnabled: true <3>
+  apiServerTimeout: 15s <4>
+  apiCheckInterval: 5s <5>
+  maxApiErrorThreshold: 3 <6>
+  peerApiServerTimeout: 5s <7>
+  peerDialTimeout: 5s <8>
+  peerRequestTimeout: 5s <9>
+  peerUpdateInterval: 15m <10>
+----
+
+<1> Specify the timeout duration for the surviving peer, after which the Operator can assume that an unhealthy node has been rebooted. The Operator automatically calculates the lower limit for this value. However, if different nodes have different watchdog timeouts, you must change this value to a higher value.
+<2> Specify the file path of the watchdog device in the nodes. If you enter an incorrect path to the watchdog device, the Self Node Remediation Operator automatically detects the softdog device path.
++
+If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a software reboot.
+<3> Specify if you want to enable software reboot of the unhealthy nodes. By default, the value of `isSoftwareRebootEnabled` is set to `true`. To disable the software reboot, set the parameter value to `false`.
+<4> Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation.
+<5> Specify the frequency to check connectivity with each API server.
+<6> Specify a threshold value. After reaching this threshold, the node starts contacting its peers.
+<7> Specify the timeout duration for the peer to connect the API server.
+<8> Specify the timeout duration for establishing connection with the peer.
+<9> Specify the timeout duration to get a response from the peer.
+<10> Specify the frequency to update peer information, such as IP address.
+
+[id="understanding-self-node-remediation-remediation-template-config_{context}"]
+== Understanding the Self Node Remediation Template configuration
+
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+
+The default remediation strategy is `NodeDeletion` that removes the `node` object.
```

Comment:
`The default remediation strategy is NodeDeletion`

From 4.11 default remediation strategy is ResourceDeletion

(And we are planing to deprecate and remove Node Deletion strategy in 4.12)
cc: @mshitrit

Reply:
- I agree that default  remediation strategy should be fixed (from  NodeDeletion to ResourceDeletion)
- Regarding commenting on NodeDeletion removed in 4.12 I'm not so sure - no strong opinion either way so I'm fine with following the usual convention for such use case.

Reply:
Thank you @prabinovRedhat and @mshitrit 
As you might know, the updates related to the default remediation strategy and the two remediation templates were reported recently and they are now being addressed as part of TELCODOCS-790. I will share the https://github.com/openshift/openshift-docs/pull/47700 for review after it is completed.